src/app/components/apparatus-entry/apparatus-entry.component.ts

```diff
@@ -16,8 +21,42 @@ export class ApparatusEntryComponent {
 
   public opened = false;
   public isInsideAppDetail: boolean;
+  public appIsInsideApp: boolean;
   public nestedApps: ApparatusEntry[] = [];
 
+  private _hd: HighlightData;
+  @Input() set highlightData(hd: HighlightData) {
+    hd.highlight = true;
+    hd.highlightColor = AppConfig.evtSettings.edition.readingColorLight;
+    this._hd = hd;
```

Comment:
If you override both `highlight` and `highlightColor`, why do you need a setter? 
Probably the setter is useful, thus you should put this logic elsewhere.